ansible/files/sonic_lab_links.csv

```diff
@@ -1,35 +1,35 @@
-StartDevice,StartPort,EndDevice,EndPort,BandWidth,VlanID,VlanMode
-str-msn2700-01,Ethernet0,str-7260-10,Ethernet1,40000,1681,Access
-str-msn2700-01,Ethernet4,str-7260-10,Ethernet2,40000,1682,Access
-str-msn2700-01,Ethernet8,str-7260-10,Ethernet3,40000,1683,Access
-str-msn2700-01,Ethernet12,str-7260-10,Ethernet4,40000,1684,Access
-str-msn2700-01,Ethernet16,str-7260-10,Ethernet5,40000,1685,Access
-str-msn2700-01,Ethernet20,str-7260-10,Ethernet6,40000,1686,Access
-str-msn2700-01,Ethernet24,str-7260-10,Ethernet7,40000,1687,Access
-str-msn2700-01,Ethernet28,str-7260-10,Ethernet8,40000,1688,Access
-str-msn2700-01,Ethernet32,str-7260-10,Ethernet9,40000,1689,Access
-str-msn2700-01,Ethernet36,str-7260-10,Ethernet10,40000,1690,Access
-str-msn2700-01,Ethernet40,str-7260-10,Ethernet11,40000,1691,Access
-str-msn2700-01,Ethernet44,str-7260-10,Ethernet12,40000,1692,Access
-str-msn2700-01,Ethernet48,str-7260-10,Ethernet13,40000,1693,Access
-str-msn2700-01,Ethernet52,str-7260-10,Ethernet14,40000,1694,Access
-str-msn2700-01,Ethernet56,str-7260-10,Ethernet15,40000,1695,Access
-str-msn2700-01,Ethernet60,str-7260-10,Ethernet16,40000,1696,Access
-str-msn2700-01,Ethernet64,str-7260-10,Ethernet17,40000,1697,Access
-str-msn2700-01,Ethernet68,str-7260-10,Ethernet18,40000,1698,Access
-str-msn2700-01,Ethernet72,str-7260-10,Ethernet19,40000,1699,Access
-str-msn2700-01,Ethernet76,str-7260-10,Ethernet20,40000,1700,Access
-str-msn2700-01,Ethernet80,str-7260-10,Ethernet21,40000,1701,Access
-str-msn2700-01,Ethernet84,str-7260-10,Ethernet22,40000,1702,Access
-str-msn2700-01,Ethernet88,str-7260-10,Ethernet23,40000,1703,Access
-str-msn2700-01,Ethernet92,str-7260-10,Ethernet24,40000,1704,Access
-str-msn2700-01,Ethernet96,str-7260-10,Ethernet25,40000,1705,Access
-str-msn2700-01,Ethernet100,str-7260-10,Ethernet26,40000,1706,Access
-str-msn2700-01,Ethernet104,str-7260-10,Ethernet27,40000,1707,Access
-str-msn2700-01,Ethernet108,str-7260-10,Ethernet28,40000,1708,Access
-str-msn2700-01,Ethernet112,str-7260-10,Ethernet29,40000,1709,Access
-str-msn2700-01,Ethernet116,str-7260-10,Ethernet30,40000,1710,Access
-str-msn2700-01,Ethernet120,str-7260-10,Ethernet31,40000,1711,Access
-str-msn2700-01,Ethernet124,str-7260-10,Ethernet32,40000,1712,Access
-str-7260-11,Ethernet19,str-acs-serv-01,p4p1,40000,,Trunk
-str-7260-11,Ethernet30,str-7260-10,Ethernet64,40000,1681-1712,Trunk
+StartDevice,StartPort,EndDevice,EndPort,BandWidth,VlanID,VlanMode,AutoNeg
```

Comment:
Recommend using on and off here as opposed to True False to keep everything in sync

Reply:
Hi @vdahiya12 when using gen mg it requires True or False, if write directly to config_db (SONiC fanout config)

Reply:
I did not quite get the gen mg part, 
Actaully in our config_db too today we have 'autoneg': 'on', so thats why I asked you to change the True False part to on or off in the column values.